x/ibc/23-commitment/types/merkle.go

```diff
@@ -65,6 +66,15 @@ func (mp MerklePrefix) IsEmpty() bool {
 	return len(mp.Bytes()) == 0
 }
 
+// PackAny converts merkle prefix to protobuf Any.
+func (mp MerklePrefix) PackAny() (*cdctypes.Any, error) {
```

Comment:
Do merkle stuff need `UnpackAny()`?

Reply:
I tried but it didn't work. I implemented the methods located in codec that unpack the proof and prefix